gitlab/resource_gitlab_project_variable.go

```diff
@@ -105,7 +105,6 @@ func resourceGitlabProjectVariableUpdate(d *schema.ResourceData, meta interface{
 	protected := d.Get("protected").(bool)
 
 	options := &gitlab.UpdateVariableOptions{
-		Key:              &key,
```

Comment:
For the record, here's the related commit https://github.com/xanzy/go-gitlab/commit/165f0f268451ae4a0c9f84e316349783bffa4771